packages/frontend/src/hooks/useColumns.tsx

```diff
@@ -59,6 +60,16 @@ export const getRawValueCell = (
     return <span>{`${raw}`}</span>;
 };
 
+export const getValueCell = (
```

Comment:
This looks like the 'formatter' method for the data. Lets abstract this out into a new file explicitly for formatting values from the warehouses, rather than having it buried in here with the `useColumns` info

Reply:
🤔 but this is only used when declaring `tanstack` table columns. Above you can see other similar methods like: `getFormattedValueCell` and `getRawValueCell`

Should I add it to be close the shared `DataViz` folder?

<img width="1298" alt="image" src="https://github.com/user-attachments/assets/be8da620-b373-4583-bb67-b43e9261a846">


(btw I can't place it in `common` because of the types and it being out of the `FE-context`)

Reply:
Oh yeah I meant something like:

```
export const getValueCell = (tanstackStuff) => {
  const value = info.getValue()
  const formatted = format(value)
   return <span>{ formatted }</span>;
}
```

That way the function is all tanstack specific. But the formatting logic lives with the `dataviz` stuff

Reply:
got itttttt